### ISSUE TITLE
A11y_Code.visualstudio.com_visual studio Learn page_Overview_Info and Relationship: While navigating with keyboard, drop down Icon is not visible, but if we hover using mouse then only we came know that we are having dropdown icon and that is a dropdown.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#5160]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG1.3.1;#DesktopWeb;#A11ySev1;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;

## Environmental Details:
#URL: https://code.visualstudio.com/learn

## OS & Browser Details :
Microsoft Windows 10 Enterprise
Version 10.0.19044 Build 19044
Microsoft Edge
Version 99.0.1150.7 (Official build) dev (64-bit)

## Repro Steps:

1. Hit the URL: https://code.visualstudio.com/learn
2. Tab till any items under overview
3. Check whether the  dropdown icon corresponding to the items is visible using keyboard or not.

## Actual results:
While navigating with keyboard, drop down Icon is not visible, but if we hover using mouse then only we are having dropdown icon and it is a dropdown.

## Expected results:
While navigating with keyboard, drop down Icon should be  visible,  then only we came know that  we are having dropdown icon and it is a dropdown.

## User Impact:
Users will get confuse to differentiate whether they are dropdowns or texts.

## Attachments:
https://user-images.githubusercontent.com/99243079/153212033-21b6db4c-bd74-4a67-9647-9a9b0c97cdf1.mp4

**Comment by rzhao271** [comment]
Duplicate of https://github.com/microsoft/vscode-docs/issues/5120

**Comment by Samhitha66** [comment]
@gregvanl 

Checked again this bug in the below environment
For main issue URL: https://code.visualstudio.com/learn

While navigating with the keyboard, dropdown icon is visible.
So,  issue is fixed


https://user-images.githubusercontent.com/99243079/157207596-3560f72b-4ffc-4f62-9ad7-618cab33d863.mp4
